app/resources/locale/en/en.po

```diff
@@ -1246,90 +1235,108 @@ msgstr "Wrong block format: {{type}}"
 msgid "Wrong block name {{name}}"
 msgstr "Wrong block name {{name}}"
 
-#: app/scripts/controllers/menu.js:1907
-#, fuzzy
+#: app/scripts/controllers/menu.js:1925
 msgid "Wrong new name!"
-msgstr "Wrong port name {{name}}"
+msgstr "Wrong new name!"
 
-#: app/scripts/services/project.js:143
+#: app/scripts/services/project.js:154
 msgid "Wrong project format: {{name}}"
 msgstr "Wrong project format: {{name}}"
 
-#: app/scripts/services/tools.js:862
+#: app/scripts/services/tools.js:973
 msgid "Wrong remote hostname {{name}}"
 msgstr "Wrong remote hostname {{name}}"
 
-#: app/scripts/controllers/menu.js:1811
+#: app/scripts/controllers/menu.js:1829
 msgid "Wrong search name!"
-msgstr ""
+msgstr "Wrong search name!"
 
-#: app/scripts/services/forms.js:412 app/views/design.html:166
+#: app/scripts/services/forms.js:412 app/views/design.html:164
 msgid "Yellow"
 msgstr "Yellow"
 
-#: app/scripts/controllers/menu.js:394
-#, fuzzy
+#: app/scripts/controllers/menu.js:409
+msgid ""
+"You are editing a submodule, if you save it, you'll save only               "
+"the submodule (in this situation \"save as\" works like               "
+"\"export module\"). Do you want to continue?"
+msgstr ""
+"You are editing a submodule, if you save it, you'll save only               "
+"the submodule (in this situation \"save as\" works like               "
+"\"export module\"). Do you want to continue?"
```

Comment:
"You are editing a submodule: If you save, you'll               "
"save only this submodule (\"save as\" works like               "
"\"export module\"). Do you want to continue?"

Reply:
👌